### DISCUSSION TITLE
access denied to Google Docs in outline page despite been authorized directly

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#4127] [General] [answered]
When a google document is included in a page, I get this message
![image](https://user-images.githubusercontent.com/80455946/190355768-8c999492-d8ac-4182-9d51-e77eb914b51b.png)

but clicking on the link "open" permits me to access it.
Could it be that I forgot to set up some configuration ?

self hosted.
outline version 0.66

**Comment by tommoor** [comment, marked as answer]
Most likely third-party cookies are blocked in the browser

**Reply by bmm-alc** [reply]
Thanks, that's it
